.github/actions/validate-access/action.yaml

```
@@ -0,0 +1,28 @@
+name: Validate Access
```

```suggestion
name: Validate Actor
```

Can you change the remaining names accordingly pls? This is more descriptive of the action cause that's what it does - check if the `actor` has the right permissions

---

Fixed.